terraform/aws/app-infrastructure/fluentbit/modules/helm-release/values.yaml

```diff
@@ -402,7 +402,6 @@ config:
         region us-east-1
         log_group_name fluent-bit-cloudwatch

```

Comment:
This name needs to come from the input variable, fluentbit_cwlogsgroup_name,  or else there could be a mismatch.